src/test/functional/pages/PimPage.ts

```diff
@@ -1,6 +1,67 @@
-import { BasePage } from "./BasePage";
+import { Locator, Page } from 'playwright';
 
+import { newEmployeeData } from '../data';
+
+import { BasePage } from './BasePage';
+import { PersonalDetailsPage } from './PersonalDetailsPage';
+
+const randomNumber = (Math.random() * 1000).toString().substring(0, 3);
```

Comment:
const randomNumber = (Math.random() * 1000).toString().substring(0 <-- this code is repeated twice (PersonalDetailsPage), let's move it to helper and add parameter substring function: substring(0, **n);**

Reply:
BTW. it doesn't return number - so the name of a function shouldn't suggest that it is ;]